CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- minor: ([21](https://github.com/meateam/permission-service/pull/21)): added updatedAt and createdAt fields, pagination sorting defaults to updated_at field
```

Comment:
Why the `minor`?
In my opinion you should link the issue instead of the pr.
Obviously, it should also be very easy to find the linked pr of the issue from the issue page.

Reply:
it's minor beecause it adds properties to the object and with that a sorting feature so it's not hotfix. It's not major because breaking changes were not added. In fact, no changes are required at all in api-gw.
as for the link - there are positive sides to the PR such as a direct link to the code changed and the time it was inserted